test/Spec.hs

```diff
@@ -11,6 +11,10 @@ import Data.Maybe (isJust, fromMaybe)
 
 main :: IO ()
 main = hspec $ do
+  describe "parse HEALTHCHECK" $
+    it "parse healthcheck without args" $
+        assertAst "HEALTHCHECK --interval=5m \\nCMD curl -f http://localhost/" [Healthcheck "--interval=5m \\nCMD curl -f http://localhost/"]
```

Comment:
Yes makes sense in a later parse. For now I just want to ensure that the parsing fails unexpectedly.